packages/braintree-integration/src/braintree-accelerated-checkout/braintree-accelerated-checkout-utils.ts

```diff
@@ -276,6 +277,7 @@ export default class BraintreeAcceleratedCheckoutUtils {
                 provider: methodId,
                 trustedShippingAddress: false,
                 type: 'card',
+                untrustedShippingCardVerificationMode: UntrustedShippingCardVerificationType.PAN,
```

Comment:
May I ask you to double check if this is not braking anything when customer uses instruments with BT AXO?